samples/nrf9160/modem_trace_flash/Kconfig.trace_flash

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+module = MODEM_TRACE_FLASH_BACKEND
+module-str = Modem trace flash backend
+source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
+
+if NRF_MODEM_LIB_TRACE
+
+# Adds NRF_MODEM_LIB_TRACE_BACKEND_CUSTOM to the trace backend choice.
```

Comment:
Remove

Reply:
I think the comment should be there to explain what it does, but I'll update it to use the actual choice `NRF_MODEM_LIB_TRACE_BACKEND_FLASH`.